public/js/controllers/game.js

```diff
@@ -1,184 +1,259 @@
-angular.module('mean.system')
-.controller('GameController', ['$scope', 'game', '$timeout', '$location', 'MakeAWishFactsService', '$dialog', function ($scope, game, $timeout, $location, MakeAWishFactsService, $dialog) {
-    $scope.hasPickedCards = false;
-    $scope.winningCardPicked = false;
-    $scope.showTable = false;
-    $scope.modalShown = false;
-    $scope.game = game;
-    $scope.pickedCards = [];
-    var makeAWishFacts = MakeAWishFactsService.getMakeAWishFacts();
-    $scope.makeAWishFact = makeAWishFacts.pop();
-
-    $scope.pickCard = function(card) {
-      if (!$scope.hasPickedCards) {
-        if ($scope.pickedCards.indexOf(card.id) < 0) {
-          $scope.pickedCards.push(card.id);
-          if (game.curQuestion.numAnswers === 1) {
-            $scope.sendPickedCards();
-            $scope.hasPickedCards = true;
-          } else if (game.curQuestion.numAnswers === 2 &&
+angular.module('mean.system') //eslint-disable-line
+  .controller('GameController', [
+    '$scope', 'game', '$timeout', '$location',
+    'MakeAWishFactsService', '$dialog',
+    '$http',
+    function (
+      $scope, game, $timeout, $location,
+      MakeAWishFactsService, $dialog, $http
+    ) {
+      $scope.hasPickedCards = false;
+      $scope.winningCardPicked = false;
+      $scope.showTable = false;
+      $scope.modalShown = false;
+      $scope.game = game;
+      // Get email address from search box
+      $scope.getUserEmail = function (user) {
+        return user.email;
+      };
+      // $scope.error = game;
+      $scope.userInfo = {};
+      $scope.messageInvite = '';
+
+      $scope.resetForm = () => {
+        $scope.user = {};
+      };
+
+      // Make request to sever if email is valid
+      $scope.getUserInfo = (email, searchForm) => {
```

Comment:
In your view, I see that the parameters you pass to this method `getUserInfo(user, searchForm)` is that of the user, why then do you have this method expecting an **email** value? Is this a typo error or is there something I am missing here? 

Please address this.